docs/framework/guides/development-tools.md

```diff
@@ -94,6 +94,49 @@ The inspector works with CKEditor 5 [v12.0.0](https://github.com/ckeditor/ckedit
 
 The source code of CKEditor 5 inspector and its issue tracker is available on GitHub in https://github.com/ckeditor/ckeditor5-inspector.
 
+## MrGit
+
+[MrGit](https://github.com/cksource/mrgit) is a Multi-repo manager for git. In CKEditor 5 it can be used for an easy development and testing of various CKEditor 5 helper repositories, such as [ckeditor5-dev](https://github.com/ckeditor/ckeditor5-dev) or [ckeditor5-linters-config](https://github.com/ckeditor/ckeditor5-linters-config).
```

Comment:
```suggestion
[Mr. Git](https://github.com/cksource/mrgit) is a multi-repo manager for git. In CKEditor 5, it can be used for easy development and testing of various CKEditor 5-related repositories, such as [ckeditor5-dev](https://github.com/ckeditor/ckeditor5-dev) or [ckeditor5-linters-config](https://github.com/ckeditor/ckeditor5-linters-config).
```